config/linux-nitropad-x.config

```diff
@@ -2746,7 +2564,7 @@ CONFIG_HARDENED_USERCOPY=y
 # CONFIG_STATIC_USERMODEHELPER is not set
 # CONFIG_IMA_SECURE_AND_OR_TRUSTED_BOOT is not set
 CONFIG_DEFAULT_SECURITY_DAC=y
-CONFIG_LSM="landlock,lockdown,yama,loadpin,safesetid,integrity,bpf"
+CONFIG_LSM="lockdown,yama,loadpin,safesetid,integrity,bpf"
```

Comment:
@daringer not sure if this change is desired but landlock is for sandboxing. Nothing uses it under Heads as of now

https://docs.kernel.org/security/landlock.html